### PR TITLE
Toolbar button for Notes

### PR DESCRIPTION
**Pull request by cedricium**
Fixes https://github.com/mozilla/notes/issues/535

Initial start at implementing toolbar button to open Notes directly. Currently, if the sidebar is closed and the Notes browser action (toolbar button) is clicked, the sidebar will open with Notes as the active page.

Refs #535 

**To-Do:**
- [x] Add active state to toolbar button if Notes is currently open (similar to sidebar toolbar button) -> not sure if possible
- [x] Add toggle ability to toolbar button:
   - if Notes is open and toolbar button is clicked, Notes/sidebar should close
   - if Notes/sidebar is closed and toolbar button is clicked, Notes should open
- [x] Get design approvals (specifically regarding toolbar button icon) from @ryanfeeley 

**Example:**
notes_toolbarbtn_demo

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -173,3 +173,8 @@ browser.storage.local.get()
       // set defaultTheme as initial theme in local storage
       browser.storage.local.set(defaultTheme);
 });
+
+// Handle onClick event for the toolbar button
+browser.browserAction.onClicked.addListener(() => {
+  browser.sidebarAction.open();
```
Comment (Natim) [inline src/background.js]: Don't we have a toggle sidebar Action ?

Comment (cedricium) [reply src/background.js]: I'm not sure what you mean - are you referring to the Sidebar browser action button that looks like this: https://design.firefox.com/icons/viewer/#sidebar?

If so, that toggles the sidebar, but will open whatever page was last opened in the sidebar (so potentially Bookmarks, History, or Notes open). This PR is supposed to add the toolbar button which opens **only** opens Notes. Let me know if that's what you meant or not.

**Comment by ryanfeeley** [comment]
👍

**Comment by johngruen** [comment]
@cedricium you can add light and dark theme icons like this: https://github.com/mozilla/speaktome/blob/master/extension/manifest.json#L495

unfortunately they'll need to be PNG.

**Comment by vladikoff** [comment]
@cedricium We are back from holiday vacations! woo! did you have time to update the icons to support dark and light themes? Otherwise we are full steam ahead to merge this fro the next release!

**Comment by vladikoff** [comment]
> unfortunately they'll need to be PNG.

@johngruen the code you linked is using svgs, updated?

**Comment by sebastienbarbier** [comment]
Update manifest to implement theming on browser_action icons.
theming-notes-icon

**Comment by Natim** [comment]
> Add active state to toolbar button if Notes is currently open (similar to sidebar toolbar button) -> not sure if possible

How would we like it to work? Should the icon turn blue when open? @ryanfeeley any idea about that?

**Comment by sebastienbarbier** [comment]
Tryed to solve active state on browserAction icon using a blue icon. Not sure if this follow any guideline, need to be discussed/defined.

toggle-notes-icon

**Comment by vladikoff** [comment]
> @vladikoff @cedricium Hope this will help, just updated manifest to implement theming on browser_action icons.

woa awesome!!

**Pull request merged**